srcjs/input_binding_radio.js

```diff
@@ -5,10 +5,23 @@ $.extend(radioInputBinding, {
   },
   getValue: function(el) {
     // Select the radio objects that have name equal to the grouping div's id
-    return $('input:radio[name="' + $escape(el.id) + '"]:checked').val();
+    var value = $('input:radio[name="' + $escape(el.id) + '"]:checked').val();
+    // If none are checked, the input will return null (it's the default on load,
+    // but it wasn't emptied when calling updateRadioButtons with character(0)
+    if (value) {
+      return value
+    } else {
+      return null
+    }
   },
   setValue: function(el, value) {
-    $('input:radio[name="' + $escape(el.id) + '"][value="' + $escape(value) + '"]').prop('checked', true);
+    if (value.length === 0){
```

Comment:
For similar reasons to the other comment, we may also want/need to handle the case that `value` is `undefined`?

Reply:
Done! I've chose to return early if value is undefined, let me know!